install-stubs/database/migrations/fill_default_admin_user_and_permissions.php

```diff
@@ -181,6 +183,10 @@ public function up()
                 if (is_null($userItem)) {
                     $userId = DB::table($this->userTable)->insertGetId($user);
 
+                    AdminUser::find($userId)->addMedia(storage_path()."/images/avatar.png")
```

Comment:
I need to mention here, that using Model in migration can take you where you do not want to go.